### PR TITLE
Update zh cn translation

### PR DESCRIPTION
**Pull request by SukkaW**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)

Update translations for zh-CN.

### CODE REVIEW

File: source/zh-cn/docs/troubleshooting.md [hunk 1]
```diff
@@ -120,3 +132,54 @@ $ echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo
...
+
+在执行 `hexo server` 后返回如下错误：
+
+```
+Error: watch /path/to/hexo/theme/ EMPERM	
+```
+
+这是由于你使用的 WSL 版本不支持监听文件系统改动。 最新版的 WSL 已经解决了这一问题。
+
+您也仍然可以通过先使用 `hexo g` 生成文件然后将其作为静态服务器运行来从WSL环境运行服务器：
```
Comment (Xeonacid) [inline source/zh-cn/docs/troubleshooting.md]:
```suggestion
您也仍然可以通过先使用 `hexo g` 生成文件然后将其作为静态服务器运行来从 WSL 环境运行服务器：
```

File: source/zh-cn/docs/troubleshooting.md [hunk 2]
```diff
@@ -120,3 +132,54 @@ $ echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo
...
+有的时候你在执行 `hexo generate` 时会返回以下错误信息：
+
+```
+FATAL Something's wrong. Maybe you can find the solution here: http://hexo.io/docs/troubleshooting.html
+Template render error: (unknown path)
+```
+
+这表明你的文件中存在一些不可被识别的字符，比如不可见的零宽度字符。有可能你的新文章存在这个问题，或者你在修改配置文件时导致了这个错误。
+
+检查你的 `_config.yml` 文件中是否漏掉了列表前的空格。你可以查阅 Wikipedia 中 [YAML](https://en.wikipedia.org/wiki/YAML) 相关页面来学习 YAML 语法。
```
Comment (Xeonacid) [inline source/zh-cn/docs/troubleshooting.md]:
```suggestion
检查你的 `_config.yml` 文件中是否漏掉了列表前的空格。你可以查阅 Wikipedia 中 [YAML](https://zh.wikipedia.org/wiki/YAML) 相关页面来学习 YAML 语法。
```

File: source/zh-cn/api/helper.md [hunk 3]
```diff
@@ -21,4 +21,19 @@ hexo.extend.helper.register('js', function(path){
...
 // <script type="text/javascript" src="script.js"></script>
-```
\ No newline at end of file
+```
+
+## FAQ
+
+### custom helper 应该放在哪里？
+
+放置在 `themes/<yourtheme>/scripts` 目录中
```
Comment (Xeonacid) [inline source/zh-cn/api/helper.md]:
```suggestion
放置在 `themes/<yourtheme>/scripts` 目录中。
```

**Comment by 1v9** [comment]
Please merge this PR 🍺

**Comment by YoshinoriN** [comment]
@SukkaW @Xeonacid @1v9 
Thank you very much 🍺 
Would you please wait a while?

@NoahDragon @JLHwung @curbengh 
Would you please check this?
I can understand this PR roughly using by google translate that it basically seems no problem.

File: source/zh-cn/api/index.md [hunk 4]
```diff
@@ -24,6 +24,7 @@ hexo.init().then(function(){
 `safe` | 开启安全模式。不加载任何插件。| `false`
 `silent` | 开启安静模式。不在终端中显示任何信息。| `false`
 `config` | 指定配置文件的路径。| `_config.yml`
+`draft` / `drafts`| 是否将草稿加入到文章列表中。<br>例如在 `hexo.locals.get('posts')` 中获取草稿内容 | _config.tml 中 `render_drafts` 的值
```
Comment (curbengh) [inline source/zh-cn/api/index.md]: prefer to use _config.yml

File: source/zh-cn/docs/commands.md [hunk 5]
```diff
@@ -24,6 +24,28 @@ $ hexo new [layout] <title>
 $ hexo new "post title with whitespace"
 ```
 
+参数 | 描述
+--- | ---
+`-p`, `--path` | 自定义新文章的路径
+`-r`, `--replace` | 如果存在同名文章，将其替换
+`-s`, `--slug` | 文章的 Slug，作为 markdown 文件的文件名和发布后的 URL
```
Comment (curbengh) [inline source/zh-cn/docs/commands.md]: not limited to markdown. prefer "文章文件" (or something similar) to "markdown 文件".

File: source/zh-cn/docs/commands.md [hunk 6]
```diff
@@ -155,11 +179,26 @@ $ hexo --silent
 
 ### 自定义配置文件的路径
 
-``` bash
-$ hexo --config custom.yml
+```bash
+# use 'custom.yml' in place of '_config.yml'
```
Comment (curbengh) [inline source/zh-cn/docs/commands.md]: comments should be translated as well.

File: source/zh-cn/docs/configuration.md [hunk 7]
```diff
@@ -41,9 +41,9 @@ title: 配置
 `tag_dir` | 标签文件夹 | `tags`
 `archive_dir` | 归档文件夹 | `archives`
 `category_dir` | 分类文件夹 | `categories`
-`code_dir` | Include code 文件夹 | `downloads/code`
+`code_dir` | Include code 文件夹，`source_dir` 下的子目录 | `downloads/code`
 `i18n_dir` | 国际化（i18n）文件夹 | `:lang`
-`skip_render` | 跳过指定文件的渲染，您可使用 [glob 表达式](https://github.com/micromatch/micromatch#extended-globbing)来匹配路径。 |
+`skip_render` | 跳过指定文件的渲染。匹配到的文件将会被不做改动的复制到 `public` 目录中。，您可使用 [glob 表达式](https://github.com/micromatch/micromatch#extended-globbing)来匹配路径。<br/><br/>例如，`skip_render: "mypage/**/*"` 将会把 `source/mypage/index.html` 和 `source/mypage/code.js` 直接复制为 `public/mypage/index.html` 和 `public/mypage/code.js`。<br/>你也可以用这种方法来跳过对指定 markdown 文件的渲染，如 `skip_render: "_posts/test-post.md"` 将不会把 `source/_posts/test-post.md` 渲染成 HTML，但 markdown 文件并不会被复制到 `public`。 |
```
Comment (curbengh) [inline source/zh-cn/docs/configuration.md]: I'm aware that the en version currently also use this format, but I suggest to move the example out of the table,

like so

https://github.com/hexojs/site/blob/master/source/docs/configuration.md#includeexclude-files-or-folders

File: source/zh-cn/docs/configuration.md [hunk 8]
```diff
@@ -98,6 +98,93 @@ Hexo 使用 [Moment.js](http://momentjs.com/) 来解析和显示时间。
...
+  # 不包括 'source/js/' 中的文件和子目录下的任何文件
+  - "js/**/*"
+  # 不包括 'source/js/' 目录下的所有文件名以 'test' 开头的文件，但包括其它文件和子目录下的单文件
+  - "js/test*"
+  # 不包括 'source/js/' 及其子目录中任何以 'test' 开头的文件
+  - "js/**/test*"
+  # 不要用 exclude 来忽略 markdown 文件。你应该使用 'skip_render'，或者在 markdown 文件的文件名之前加一个 '_'
+  # 如果你在这里配置一个 - "_posts/hello-world.md" 是没有用的。
+```
+
```
Comment (curbengh) [inline source/zh-cn/docs/configuration.md]: please include translation of "Each value in the list must be enclosed with single/double quotes." as well.

File: source/zh-tw/docs/tag-plugins.md [hunk 9]
```diff
@@ -187,7 +187,7 @@ content
 
 ## Include Code
 
-插入 `source` 資料夾內的程式檔。
+插入 `source/downloads/code` 資料夾內的程式檔。
```
Comment (curbengh) [inline source/zh-tw/docs/tag-plugins.md]: would be clearer if also include "...。資料夾取决于你在配置文件中 `code_dir` 的配置。"

I will update en version of this line.

File: source/zh-cn/api/index.md (hunk 4 above)
Comment (SukkaW) [reply source/zh-cn/api/index.md]: Fixed in commit https://github.com/hexojs/site/pull/1030/commits/04ff7bb7c12f48eb3b6e4f2a9739d015002328fe

File: source/zh-cn/docs/commands.md (hunk 5 above)
Comment (SukkaW) [reply source/zh-cn/docs/commands.md]: Fixed in commit https://github.com/hexojs/site/pull/1030/commits/04ff7bb7c12f48eb3b6e4f2a9739d015002328fe

File: source/zh-cn/docs/commands.md (hunk 6 above)
Comment (SukkaW) [reply source/zh-cn/docs/commands.md]: The translations of those comments were added in https://github.com/hexojs/site/pull/1030/commits/04ff7bb7c12f48eb3b6e4f2a9739d015002328fe

File: source/zh-cn/docs/configuration.md (hunk 7 above)
Comment (SukkaW) [reply source/zh-cn/docs/configuration.md]: The example has now moved out of the table in https://github.com/hexojs/site/commit/04ff7bb7c12f48eb3b6e4f2a9739d015002328fe, just as #1033.

File: source/zh-cn/docs/configuration.md (hunk 8 above)
Comment (SukkaW) [reply source/zh-cn/docs/configuration.md]: The missing translation were added in commit https://github.com/hexojs/site/commit/04ff7bb7c12f48eb3b6e4f2a9739d015002328fe

File: source/zh-tw/docs/tag-plugins.md (hunk 9 above)
Comment (SukkaW) [reply source/zh-tw/docs/tag-plugins.md]: Added translations of this line for zh-tw in https://github.com/hexojs/site/commit/04ff7bb7c12f48eb3b6e4f2a9739d015002328fe

File: source/zh-cn/docs/commands.md [hunk 10]
```diff
@@ -24,6 +24,28 @@ $ hexo new [layout] <title>
 $ hexo new "post title with whitespace"
 ```
 
+参数 | 描述
+--- | ---
+`-p`, `--path` | 自定义新文章的路径
+`-r`, `--replace` | 如果存在同名文章，将其替换
+`-s`, `--slug` | 文章的 Slug，作为新文章的文件名和发布后的 URL
+
+默认情况下，Hexo 会使用文章的标题来决定 markdown 文件的路径。对于独立页面来说，Hexo 会创建一个以标题为名字的目录，并在目录中放置一个 `index.md` 文件。你可以使用 `--path` 参数来覆盖上述行为、自行决定文件的目录：
```
Comment (curbengh) [inline source/zh-cn/docs/commands.md]: you missed this line in rewriting "markdown 文件".

Comment (SukkaW) [reply source/zh-cn/docs/commands.md]: Oops.. I will fix it.

Comment (SukkaW) [reply source/zh-cn/docs/commands.md]: All the `markdown 文件` has been rewritten in `文章文件` in https://github.com/hexojs/site/pull/1030/commits/602ad8119a690642a30ddb13e003f7e9af044510

**Comment by curbengh** [comment]
@Xeonacid 
anything you would like to add?

**Pull request merged**